src/lib/libraries/decks/translate-image.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview
+ * Utility functions for handling tutorial images in multiple languages
+ */
+
+import {enImages as defaultImages} from './en-steps.js';
+
+let savedImages = {};
+let savedLocale = '';
+
+const loadSpanish = () =>
+    import(/* webpackChunkName: "es-steps" */ './es-steps.js')
+        .then(({esImages: imageData}) => imageData);
+
+const translations = {
+    es: () => loadSpanish()
+};
+
+const loadImageData = locale => {
+    if (translations.hasOwnProperty(locale)) {
+        translations[locale]()
+            .then(newImages => {
+                savedImages = newImages;
+                savedLocale = locale;
```

Comment:
This is usually something we do in redux. I know that it works, but just noting it now in case we see issues. I'm not sure exactly what a bug from this would look like, maybe a race condition?